src/pymor/algorithms/newton.py

```diff
@@ -124,10 +130,19 @@ def newton(operator, rhs, initial_guess=None, mu=None, error_norm=None, least_sq
             correction = jacobian.apply_inverse(residual, least_squares=least_squares)
         except InversionError:
             raise NewtonError('Could not invert jacobian')
-        U.axpy(relax, correction)
+        if relax == 'armijo':
```

Comment:
Maybe better:

```python
if isinstance(relax, Number):
    ...
elif relax == 'armijo':
    ....
else:
    raise ValueError('Unknown line search method')
```
to not fail with a weird error message when the user has a typo in 'armijo' ..